README.md

```diff
@@ -1,16 +1,166 @@
-![LUSID_by_Finbourne](https://content.finbourne.com/LUSID_repo.png)
+# LUSID<sup>®</sup> Python SDK
+This is the Python SDK for [LUSID by FINBOURNE](https://www.finbourne.com/lusid-technology), a bi-temporal investment management data platform with portfolio accounting capabilities. To use it you'll need a LUSID account. [Sign up for free at lusid.com](https://www.lusid.com/app/signup)
 
-| branch | status |
-| --- | --- |
-| `master` | ![PyPI](https://img.shields.io/pypi/v/lusid-sdk?color=blue) ![run-sdk-tests](https://github.com/finbourne/lusid-sdk-python/workflows/run-sdk-tests/badge.svg?branch=master) [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=finbourne_lusid-sdk-python&metric=alert_status)](https://sonarcloud.io/dashboard?id=finbourne_lusid-sdk-python) |
-| `develop` | ![run-sdk-tests](https://github.com/finbourne/lusid-sdk-python/workflows/run-sdk-tests/badge.svg?branch=develop) |
+<a href="https://www.lusid.com/app/signup"><img src="https://content.finbourne.com/LUSID_repo.png" alt="LUSID_by_Finbourne" width="250"></a>
 
-# LUSID<sup>®</sup> Python SDK
+## Installation
 
 The PyPi package for the LUSID SDK can installed using the following:
 
 ```
-$ pip install lusid-sdk
+pip install lusid-sdk
+```
+
+We publish two versions of the Python SDK:
+
+* **lusid-sdk-python (this one) - supports `Production` and `Early Access` API endpoints**
+* [lusid-sdk-python-preview](https://github.com/finbourne/lusid-sdk-python-preview) - supports `Production`, `Early Access`, `Beta` and `Experimental` API endpoints.
+
+For more details on API endpoint categories, see [What is the LUSID feature release lifecycle?](https://support.lusid.com/knowledgebase/article/KA-01786/en-us).
+To find out which category an API endpoint falls into, see [LUSID API Documentation](https://www.lusid.com/api/swagger/index.html).
+
+> If you install both `lusid-sdk-python` and `lusid-sdk-python-preview`, `import lusid` will refer to `lusid-sdk-python`. 
+
+## Usage
+
+First, import the following modules:
+
+
+```python
+import lusid
+import lusid.models as models
+from lusid.utilities import ApiConfigurationLoader
 ```
 
-For further documentation on building the SDK, running the tutorials and using the SDK please see the [wiki](https://github.com/finbourne/lusid-sdk-python/wiki).
+Then construct the API factory:
+
+```python
+secrets_file_path = "/path/to/secrets.json"
+config = ApiConfigurationLoader.load(secrets_file_path)
+api_factory = lusid.utilities.ApiClientFactory(
+    token=lusid.utilities.RefreshingToken(config),
+    api_secrets_filename=secrets_file_path
+)
+```
+
+Now that the API client is ready, you are ready to use the various API endpoints.
+
+### List API endpoints
+
+```python
+[api for api in dir(lusid.api) if "API" in api]
+```
+
+You can construct an API endpoint by calling `api_factory.build(lusid.api.<className>)` for any of the returned classes.
+
+### Create portfolio
```

Comment:
is it worth linking to full fledged sdk examples too?

Reply:
Do you mean this directory? https://github.com/finbourne/lusid-sdk-python/tree/master/sdk/tests/tutorials

We can put a link for that in, yup.

Separate to this PR I want to write some guides that use the code from the SDK examples.